playground/mocks/data/idp/idx/enroll-profile-new.json

```diff
@@ -18,20 +18,88 @@
             "name": "userProfile",
             "form": {
               "value": [
-                {
-                  "name": "lastName",
-                  "label": "Last name",
-                  "required": true
-                },
                 {
                   "name": "firstName",
+                  "type": "string",
                   "label": "First name",
-                  "required": true
+                  "required": true,
```

Comment:
instead of altering this mock file.. I would suggest to add a new one..

enroll-profile-new-with-timezone

Reply:
good points! but there is no test method to verify the dropdown list values or form field type, so I am not sure what tests I can add for the new json. Therefore, I just only display the field in the view by appending the existing json file.

Reply:
> no test method to verify the dropdown list values
 
sorry i didn't get that..

Reply:
new mock file added